ofl/notosansphags-pa/METADATA.pb

```diff
@@ -0,0 +1,23 @@
+name: "Noto Sans Phags-Pa"
```

Comment:
@simoncozens the hyphen in family name is gonna cause trouble. The API serves this name: `Noto Sans PhagsPa` (ID1) and display name `Noto Sans Phags Pa`

Reply:
I suggest name ID 1 `Noto Sans PhagsPa` and display name `Noto Sans Phags-Pa`

Reply:
@simoncozens ping for Noto Sans Phags Pa naming